app/models/public_body.rb

```diff
@@ -591,8 +591,8 @@ def json_for_api
       # information
       # :version, :last_edit_editor, :last_edit_comment
       :home_page => calculated_home_page,
-      :notes => notes,
-      :publication_scheme => publication_scheme,
+      :notes => notes.to_s,
+      :publication_scheme => publication_scheme.to_s,
```

Comment:
Fixed by combo of extra migration to set defaults and using a persisted factory instance rather than using `.new` which doesn't really test real-world use and produces null values